src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    public static final int MAX_SIZE = 100;
```

Comment:
why is this constant here

Reply:
I thought about it as a maximum dimension for a figure and so to be something related to Figure class.
Ok, moved to the supplier. Thank you!